gamestonk_terminal/backtesting/bt_view.py

```diff
@@ -0,0 +1,288 @@
+"""bt view module"""
+__docformat__ = "numpy"
+import argparse
+from typing import List, Union
+from datetime import datetime
+import matplotlib.pyplot as plt
+import pandas as pd
+import bt
+import pandas_ta as ta
+from gamestonk_terminal.helper_funcs import parse_known_args_and_warn, plot_autoscale
+from gamestonk_terminal.config_plot import PLOT_DPI
+from gamestonk_terminal import feature_flags as gtff
+from gamestonk_terminal.backtesting.bt_helper import buy_and_hold
+
+
+def simple_ema(ticker: str, start_date: Union[str, datetime], other_args: List[str]):
+
+    parser = argparse.ArgumentParser(add_help=False, prog="ema")
+    parser.add_argument(
+        "-l", default=20, dest="length", type=int, help="EMA period to consider"
+    )
+
+    parser.add_argument(
+        "--spy",
+        action="store_true",
+        default=False,
+        help="Flag to add spy hold comparison",
+        dest="spy",
+    )
+
+    parser.add_argument(
+        "--no_bench",
+        action="store_true",
+        default=False,
+        help="Flag to not show buy and hold comparison",
+        dest="no_bench",
+    )
+
+    try:
+        ns_parser = parse_known_args_and_warn(parser, other_args)
+        if not ns_parser:
+            return
+        ticker = ticker.lower()
+        ema = pd.DataFrame()
+        prices = bt.get(ticker, start=start_date)
+        ema[ticker] = ta.ema(prices[ticker], ns_parser.length)
+        bt_strategy = bt.Strategy(
+            "AboveEMA",
+            [
+                bt.algos.SelectWhere(prices >= ema),
+                bt.algos.WeighEqually(),
+                bt.algos.Rebalance(),
+            ],
+        )
+        bt_backtest = bt.Backtest(bt_strategy, prices)
+
+        if ns_parser.spy:
+            spy_bt = buy_and_hold("spy", start_date, "SPY Hold")
+            if ns_parser.no_bench:
+                res = bt.run(bt_backtest, spy_bt)
+            else:
+                stock_bt = buy_and_hold(ticker, start_date, ticker.upper() + " Hold")
+                res = bt.run(bt_backtest, spy_bt, stock_bt)
+        else:
+            if ns_parser.no_bench:
+                res = bt.run(bt_backtest)
+            else:
+                stock_bt = buy_and_hold(ticker, start_date, ticker.upper() + " Hold")
+                res = bt.run(bt_backtest, stock_bt)
+
+        _, ax = plt.subplots(figsize=plot_autoscale(), dpi=PLOT_DPI)
```

Comment:
probably have this data on a `plot_backtest` function to follow MVC design.

Same for others